xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/editors/dataeditors.js

```diff
@@ -750,15 +750,43 @@ editors.XDataEditors = Class.create({
     });
     // Create and insert move button
     element.select('.xproperty-title .tools').each(function(item) {
-      var movebutton = new Element('span', {
-        'class': 'tool move',
+      var movebutton = new Element('button', {
+        'class': 'btn btn-secondary btn-xs tool move',
         title: $jsontool.serialize($services.localization.render('core.editors.class.moveProperty.handle.label'))
-      }).update($jsontool.serialize($services.icon.renderHTML('reposition')));
+      }).update('<span class="sr-only">' +
+        $jsontool.serialize($services.localization.render('core.editors.class.moveProperty.handle.label')) +
+        '</span>' +
+        $jsontool.serialize($services.icon.renderHTML('reposition')));
       item.makePositioned();
       item.appendChild(movebutton);
       movebutton.observe('click', function(event) {
         event.stop();
       }.bindAsEventListener());
+      // Extend the Sortable items with arrow keys support
+      movebutton.observe('keydown', function(event){
+        if (![38, 40].includes(event.keyCode)) { return; }
```

Comment:
Worth a comment to indicate what 38/40 are (Up/Down I guess?)

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3254/commits/ebc45866813fa2cd3c1aad6c0042c92862209911 :+1: